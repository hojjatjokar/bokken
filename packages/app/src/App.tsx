import React from 'react';
import { Button } from '@bokken/components';
import { sum } from '@bokken/utils';
//import Icon from './assets/images/bokken.svg';
// import './style.css';

const App = (): JSX.Element => {
  console.log('sum()', sum(1, 2, 3));
  return (
    <div>
      <h1>Hello Bokken</h1>
      <Button label="hojjat" />
    </div>
  );
};

export default App;
