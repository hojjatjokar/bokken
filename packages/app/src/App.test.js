import React from 'react';
import { render, cleanup } from '@testing-library/react';
import App from './App';

afterEach(cleanup);

describe('App', () => {
  test('renders App component', () => {
    render(<App />);
  });
  it('should take a snapshot', () => {
    const { asFragment } = render(<App />);

    expect(asFragment(<App />)).toMatchSnapshot();
  });
});
