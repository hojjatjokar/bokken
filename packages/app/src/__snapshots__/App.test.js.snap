// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should take a snapshot 1`] = `
<DocumentFragment>
  <div>
    <h1>
      Hello Bokken
    </h1>
    <button
      class="storybook-button storybook-button--medium storybook-button--secondary"
      style="background-color: green;"
      type="button"
    >
      hojjat
    </button>
  </div>
</DocumentFragment>
`;
